WindowsImgur.Core/Services/Helper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Snippur.Core.Services
+{
+    public class Helper

```

Comment:
Would prefer if this helper class's name was more descriptive of it's function.   "Helper" means it could be responsible for literally anything.